src/redux/actionTypes.js

```diff
@@ -4,5 +4,8 @@ export const actionTypes = {
     TOGGLE_POKEMON_MODAL_WINDOW: 'TOGGLE_POKEMON_MODAL_WINDOW',
     MAKE_API_REQUEST: 'MAKE_API_REQUEST',
     SELECT_POKEMON: 'SELECT_POKEMON',
-    TOGGLE_DOWNLOAD_POKEMON_MODAL_WINDOW: 'TOGGLE_DOWNLOAD_POKEMON_MODAL_WINDOW'
+    TOGGLE_DOWNLOAD_POKEMON_MODAL_WINDOW: 'TOGGLE_DOWNLOAD_POKEMON_MODAL_WINDOW',
```

Comment:
probably you don't need this action

Reply:
Unfortunately, it's needed. But I've found other actions, which I don't need.